packages/renderer/src/lib/image/ImageDetailsFiles.svelte

```diff
@@ -11,16 +11,17 @@ import FilesystemLayerView from './FilesystemLayerView.svelte';
 import { type ImageFilesystemLayerUI, toImageFilesystemLayerUIs } from './imageDetailsFiles';
 import ImageDetailsFilesLayers from './ImageDetailsFilesLayers.svelte';
 
-export let imageInfo: ImageInfo | undefined;
+let { imageInfo }: { imageInfo: ImageInfo | undefined } = $props();
+
+let imageLayers = $state<ImageFilesystemLayers>();
```

Comment:
This is wrong now ? as the real type is `ImageFilesystemLayers | undefined` ??

Reply:
It seems to be a supported notation (and the one used in other components of podman desktop). Here, `$state<ImageFilesystemLayers>()` returns a value of type `ImageFilesystemLayers | undefined`